lblod/ember-rdfa-editor-lblod-plugins
# GN-4442: Use new findNodePosUp/Down functionality to move Template Comments over whole document

### Overview
Uses `findNodePosUp/Down` to move template comments in a more clean (and general) way. The old implementation barely works.

##### connected issues and PRs:
- https://binnenland.atlassian.net/browse/GN-4442
- need https://github.com/lblod/ember-rdfa-editor/pull/939


### Setup
- checkout https://github.com/lblod/ember-rdfa-editor/pull/939
- `npm link` in editor-repo
- `npm link @lblod/ember-rdfa-editor` in plugins repo
- might need to do `ember ts:precompile` for types

### How to test/reproduce
- include a template comment in some structure in Regulatory Statements part.
- add some structured data (title, article, paragraphs ...)
- click on the template comment and move it up and down, see that this is always possible and it does not skip valid positions.
- It will also not move inside another template comment (even though, content-wise, this would be valid).

- It should never split any nodes
- Should keep the selection inside the template comment intact

### Challenges/uncertainties
note that this PR was changed (algorithm moved to editor-repo). Hence why some of the comments might not fit.
